### DISCUSSION TITLE
Connecting to sql server

### DISCUSSION THREAD

**Discussion by GabrielKNdara** [dbeaver/dbeaver#19400] [Q&A]
### Description

Hi guys .

I'm new to Dbeaver and mx linux. I want to connect dbeaver to  mssql server, yet i don't how, please help.
![Screenshot_2022-11-22_06-49-55](https://user-images.githubusercontent.com/52117833/203224824-8466444a-de62-46a3-9373-be70783b4d19.png)

### DBeaver Version

community edition 21.0.5

### Operating System

mx linux 21

### Database and driver

ms sql server

### Steps to reproduce

1. Open dbeaver community.
2. On the toolbar click on database
3. under database click on new database connection
4. Inside the dialog select which state "Select your database"
5. select Microsoft Sql Server
6. SQL Server Connection Settings dialog appears
7. This where i need help how to set up the connection.
8. Thanks in advance

**Comment by LonwoLonwo** [comment]
Hello @ProgrammerGabs 

Perhaps you can not use Windows Authentication for your Linux machine.
You can use SQL Server Authentication with the login and password. SQL Server localhost database usually have user 'sa' as default.
And usually, you need to use [SQL Sever configuration manager](https://learn.microsoft.com/en-us/sql/relational-databases/sql-server-configuration-manager?view=sql-server-ver16) to configure your connection settings.

**Comment by JohnMCouch** [comment]
Has anyone solved this? We are trying to connect to SQL Server from MAC and it is a domain laptop but fails saying untrusted device. We do not allow users to use SQL authentication to connect.

**Comment by E1izabeth** [comment]
Probably it's a duplicate of #9751

**Comment by JohnMCouchAIP** [comment]
It is. And I figured it out based on other peoples posts which also look similar.

**Comment by GabrielKNdara** [comment]
It's not solved, if you found a solution please share

**Comment by JohnMCouchAIP** [comment]
You have to use NTLM for authentication. My instructions include setting application Intent as we use AOAG and the developers connect to secondary's for queries.

**Comment by E1izabeth** [comment]
Maybe some comments here #3320 might help you.
